# HG changeset patch
# User David Bateman <dbateman@free.fr>
# Date 1227520673 -3600
#      Mon Nov 24 10:57:53 2008 +0100
# Node ID fa9e6619fa9949a96e2cd393a53a97d4c46b37ac
# Parent  18c4ded8612a66669ecc280030651a204007583e
Add Jarko Kaleva to contributors file

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -80,16 +80,17 @@ Cyril Humbert
 Teemu Ikonen
 Alan W. Irwin
 Geoff Jacobsen
 Mats Jansson
 Cai Jianming
 Steven G. Johnson
 Heikki Junes
 Atsushi Kajita
+Jarkko Kaleva
 Mohamed Kamoun
 Lute Kamstra
 Thomas Kasper
 Joel Keay
 Mumit Khan
 Paul Kienzle
 Aaron A. King
 Arno J. Klaassen
